src/views/HeaderBackButton.js

```diff
@@ -74,6 +81,11 @@ const styles = StyleSheet.create({
       resizeMode: 'contain',
       transform: [{ scaleX: I18nManager.isRTL ? -1 : 1 }],
     },
+  iconWithTitle: Platform.OS === 'ios'
+    ? {
+      marginRight: 5,
+    }
+    : {},
```

Comment:
Just like before :) We do not render title on Android at all and so, tweaks because of that are not needed. Not sure if that's what you were asking about?